wallet/tickets.go

```diff
@@ -12,6 +12,14 @@ import (
 	"github.com/decred/dcrwallet/walletdb"
 )
 
+// TicketVoteBitsUpdate is a struct containing updates to ticket vote bits
+// settings in the stake database.
+type TicketVoteBitsUpdate struct {
```

Comment:
Thinking something like this:

```
type VoteBits struct {
    VoteBits uint16
    ExtendedVoteBits []byte
}

func (w *Wallet) VoteBitsForTicket(ticketHash *chainhash.Hash) (VoteBits, error) { ... }

func (w *Wallet) SetVoteBitsForTicket(ticketHash *chainhash.Hash, voteBits VoteBits) error { ... }
```

I don't see `SetVoteBitsForTickets` (the plural one) being used anywhere so I can't make any comments on the best api there.  Unfortunately since go doesn't have first class iterators it would be rather impossible to write an API that works well there for any data structure the caller is using. :(